modules/post/multi/manage/screenshare.rb

```diff
@@ -0,0 +1,163 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info={})
+    super( update_info( info,
+      'Name'           => 'Multi Manage the screen of the target meterpreter session',
+      'Description'    => %q{
+        This module allows you to view and control the screen of the target computer via
+        a local browser window. The module continually screenshots the target screen and
+        also relays all mouse and keyboard events to session.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         => [ 'timwr'],
+      'Platform'       => [ 'linux', 'win' ],
+      'SessionTypes'   => [ 'meterpreter' ],
+      'DefaultOptions' => { 'SRVHOST' => '127.0.0.1' },
```

Comment:
I'm not 100% sure what we can do about https://en.wikipedia.org/wiki/DNS_rebinding